features/step_definitions/email_steps.rb

```diff
@@ -86,5 +86,12 @@ def expect_email_exists(message: nil, email: nil, link: nil, link_text: nil)
   Rake.application = @rake
   Rake.application.rake_require "tasks/emails"
   Rake::Task.define_task(:environment)
+  # Stub 'puts' to send import messages to the file test.log
+  IO.any_instance.stub(:puts) do |stream, msgs|
```

Comment:
Lint/UnusedBlockArgument: Unused block argument - `stream`. If it's necessary, use `_` or `_stream` as an argument name to indicate that it won't be used. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)